tools/ttnn-standalone/CMakeLists.txt

```diff
@@ -82,8 +82,11 @@ target_link_directories(ttnn-standalone PRIVATE
 target_link_libraries(ttnn-standalone PRIVATE
     # Metalium
     tt_metal
+    device
     yaml-cpp
     pthread
+    # nng
```

Comment:
probably not needed?

Reply:
So these libs were added elsewhere in our codebase when metal was uplifted. We didn't need them for the standalone, but I wanted to leave them there as comments in case the need arises, as I expect it would save a lot of time. I'd like to leave them in, if you're okay with that.

Reply:
Yep, I am ok with that. Can you please write this statement in a comment to add context for folks who will see this without this knowledge... :)